### DISCUSSION TITLE
Jetpack Compose and Kotlin Functionality

### DISCUSSION THREAD

**Discussion by zachschm** [Spikeysanju/MotionToast#62] [Q&A]
I would love to use this toast library in my school project. I am using jetpack compose and Kotlin for my app and I am not sure how or if it is possible to use with my set up.

**Comment by zachschm** [comment]
@Composable
fun sucessMessage(message: String)
{
    val context = LocalContext.current
    MotionToast.createColorToast(
        context as Activity,
        "Hurray success 😍",
        message,
        MotionToastStyle.SUCCESS,
        MotionToast.GRAVITY_BOTTOM,
        MotionToast.LONG_DURATION,
        ResourcesCompat.getFont(context,R.font.poppins))
} 

-------

I was able to get the toast to work using the above code. 

you just have to change "this" to get the context which is the new way of getting the activity

**Reply by Stalkerfish** [reply]
But how to import the project in the build.gradle.kts ?
